src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -123,6 +126,50 @@ uct_cuda_ipc_iface_flush(uct_iface_h tl_iface, unsigned flags,
     return UCS_INPROGRESS;
 }
 
+static ucs_status_t uct_cuda_ipc_iface_event_fd_get(uct_iface_h tl_iface, int *fd_p)
+{
+    uct_cuda_ipc_iface_t *iface = ucs_derived_of(tl_iface, uct_cuda_ipc_iface_t);
+
+    *fd_p = iface->eventfd;
```

Comment:
could we create the eventfd here (on first call)? so the fd would not be created and not used if not needed